jobqueue/server.go

```diff
@@ -1223,10 +1223,10 @@ func (s *Server) createQueue() {
 	// package will only call this once at a time, so we don't need to worry
 	// about locking across the whole function.
 	q.SetReadyAddedCallback(func(queuename string, allitemdata []interface{}) {
-		defer internal.LogPanic(s.Logger, "jobqueue ready added callback", true)
+		defer internal.LogPanic(ctx, "jobqueue ready added callback", true)
 
-		s.Debug("rac started")
-		defer s.Debug("rac finished")
+		clog.Error(ctx, "rac started")
```

Comment:
If it was s.Debug, then it should be clog.Debug. Double-check all replacements in all files; there's at least a few more of these mis-translations below.